### DISCUSSION TITLE
New Feature: Birthdays

### DISCUSSION THREAD

**Discussion by EmilioGerdez** [Golang-Venezuela/adan-bot#12] [General]
Greetings, 
just wanna let you all know that soon I will be making a pull request to add the birthday's feature,
mainly I want it to associate a birthday's date to the telegram members "@" so it can congratulate them,
let me know if you want anything specific to be included in this feature

**Comment by Euler-B** [comment]
This functionality will be part of a feature that we are considering implementing called **Event-Calendar.**
